packages/clarity-native-bin/postInstallScript.js

```diff
@@ -54,5 +54,5 @@ function tsNodeInstall() {
   const directInstallTsFile = path.join(__dirname, "src", "directInstall.ts");
   const tsNodeExecArgs = [tsNodePkg, "--project", tsConfigBuildFile, directInstallTsFile];
   console.log(`Running: npx ${tsNodeExecArgs.join(" ")}`);
-  childProcess.execFileSync("npx", tsNodeExecArgs, { stdio: ['pipe', process.stdout, process.stderr] });
+  childProcess.execFileSync("npx", tsNodeExecArgs, { stdio: ['pipe', process.stdout, process.stderr], shell: process.platform=="win32" });
```

Comment:
What would be the side effect of the flag `shell: process.platform=="win32"` when running on another distribution (linux, mac, ...)?

Reply:
Default is false and I don't think this is overridden with env vars, so should be okay. Tested on macOS